packages/ui-extensions/src/surfaces/admin/extension-targets.ts

```diff
@@ -171,10 +171,101 @@ export interface ExtensionTargets {
+    ActionExtensionApi<'admin.discount-index.action.render'>,
+    AllComponents
+  >;
+
+  /**
+   * Renders an admin action in the collection details page. Open this extension from the "More Actions" menu.
+   *
+   * See the [list of available components](/docs/api/admin-extensions/components).
+   */
+  'admin.collection-details.action.render': RenderExtension<
+    ActionExtensionApi<'admin.collection-details.action.render'>,
+    AllComponents
+  >;
+
+  /**
+   * Renders an admin action in the collection index page. Open this extension from the "More Actions" menu.
+   *
+   * See the [list of available components](/docs/api/admin-extensions/components).
+   */
+  'admin.collection-index.action.render': RenderExtension<
+    ActionExtensionApi<'admin.collection-index.action.render'>,
+    AllComponents
+  >;
+
+  /**
+   * Renders an admin action in the abandonded checkout page. Open this extension from the "More Actions" menu.
+   *
+   * See the [list of available components](/docs/api/admin-extensions/components).
+   */
+  'admin.abandoned-checkout-details.action.render': RenderExtension<
+    ActionExtensionApi<'admin.abandoned-checkout-details.action.render'>,
+    AllComponents
+  >;
+
+  /**
+   * Renders an admin action in the product variant details page. Open this extension from the "More Actions" menu.
+   *
+   * See the [list of available components](/docs/api/admin-extensions/components).
+   */
+  'admin.variant-details.action.render': RenderExtension<
+    ActionExtensionApi<'admin.variant-details.action.render'>,
+    AllComponents
+  >;
+
+  /**
+   * Renders an admin action in the draft order details page. Open this extension from the "More Actions" menu.
+   *
+   * See the [list of available components](/docs/api/admin-extensions/components).
+   */
+  'admin.draft-order-details.action.render': RenderExtension<
+    ActionExtensionApi<'admin.draft-order-details.action.render'>,
+    AllComponents
+  >;
+
+  /**
+   * Renders an admin action in the draft orders page. Open this extension from the "More Actions" menu.
+   *
+   * See the [list of available components](/docs/api/admin-extensions/components).
+   */
+  'admin.draft-order-index.action.render': RenderExtension<
+    ActionExtensionApi<'admin.draft-order-index.action.render'>,
+    AllComponents
+  >;
+
+  /**
+   * Renders an admin action in the discount details page. Open this extension from the "More Actions" menu.
+   *
+   * See the [list of available components](/docs/api/admin-extensions/components).
+   */
+  'admin.discount-details.action.render': RenderExtension<
+    ActionExtensionApi<'admin.discount-details.action.render'>,
+    AllComponents
+  >;
+
+  /**
+   * Renders an admin action in the order fulfilled card. Open this extension from the "3-dot" menu inside the order fulfilled card.
+   * Note: This extension will only be visible in orders which were fulfilled by the same app as the extension.
```

Comment:
Maybe second person helps here?

 This extension will only be visible on orders which were fulfilled by your app.